### DISCUSSION TITLE
Changing mouse events triggering plane widget

### DISCUSSION THREAD

**Discussion by machur** [pyvista/pyvista#5268] [Q&A]
Hi, I'm working on a viewer for high resolution meshes. I would like to wrap the meshes with plane widget, but I have some issues with intuitive user interface. I want to maintain the default plotter interaction with rendered objects, i.e., left button responsible for rotation etc. 

The plots are high res and they are usually extremely zoomed-in by the users to investigate the details. With the default action buttons of plane widgets, the plotter rotation doesn't work any longer - it is overriden with the plane widget actionos. Is there a way to assign different mouse events to the plane widget actions, e.g., left mouse button for plotter rotation of the whole coordinate system, right mouse button for dragging plane widget's plane instead?

**Comment by banesullivan** [comment]
This is an excellent question and I honestly don't know off hand. I may point you the VTK discourse to ask this question: https://discourse.vtk.org/

Once we figure out how to do this in VTK itself, implementing with PyVista should be straightforward

**Comment by machur** [comment]
@banesullivan it looks like there's an API in VTK for that https://vtk.org/doc/release/5.6/html/a00907.html:

![image](https://github.com/pyvista/pyvista/assets/4650515/086d015a-2420-44fb-94ce-d8de1128c7c9)

I haven't seen this in action, but the code looks promising.
